packages/common/src/operations.graphql

```diff
@@ -159,41 +92,20 @@ query locations($sort: [LocationSortInput!]) {
     __typename
     ... on LocationConnector {
       __typename
+      totalCount
       nodes {
         __typename
         id
         name
         onHold
         code
       }
-      totalCount
-    }
-    ... on ConnectorError {
-      __typename
-      error {
-        description
-        ... on DatabaseError {
-          __typename
-          description
-          fullError
-        }
-        ... on PaginationError {
-          __typename
-          description
-          rangeError {
-            description
-            field
-            max
-            min
-          }
-        }
-      }
     }
   }
 }
 
-mutation insertLocation($input: InsertLocationInput!) {
-  insertLocation(input: $input) {
+mutation insertLocation($input: InsertLocationInput!, $storeId: String!) {
```

Comment:
couple of storeIDs added, but not for everything. I believe that's coming a bit later.